distribution/build.gradle

```diff
@@ -40,6 +41,20 @@ buildscript {
 }
 
 /*****************************************************************************
+ *                                Notice file                                *
+ *****************************************************************************/
+
+// integ test zip only uses core, so a different notice file is needed there
+task buildCoreNotice(type: NoticeTask) {
+  dependencies project(':core')
+}
+
+// other distributions include notices from modules as well, which are added below later
+task buildFullNotice(type: NoticeTask) {
+  dependencies project(':core')
```

Comment:
I tried explaining it in the comment above, but I guess it was too terse.  Integ test zip does not have modules, so I generate a notice just with deps from core for that, while the others have the "full" notice file, which includes both core, and modules dependencies.